### PR TITLE
Improve documentation of "create tag" API

### LINKED ISSUE #7103: [BUG]: Python SDK create tag returns conflict error
"The code that is giving me issues is:"

```
tag_creation=client.models.tag_creation.TagCreation(id='commit_id',ref='ref_example')
thread=client.tags_api.create_tag("repo-name",tag_creation)
```

### PR DESCRIPTION
**Pull request by arielshaqed**
Closes #7103.

### CODE REVIEW

**Review by N-o-Z — CHANGES_REQUESTED** [review]
The ref description felt a little bit confusing

File: api/swagger.yml [hunk 1]
```diff
@@ -585,14 +585,17 @@ components:
...
       required:
         - id
         - ref
       properties:
         id:
           type: string
+          description: ID of tag to create
         ref:
           type: string
+          description: reference to commit to tag
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply api/swagger.yml]: Changed to something shorter & I hope better.

**Review by arielshaqed — COMMENTED** [review]
Thanks!  PTAL...

**Review by N-o-Z — APPROVED** [review]

**Comment by arielshaqed** [comment]
Thanks!

[J. L. Picard: Enable autosquish!](https://imgflip.com/i/88uoim)

**Pull request merged**